velox/common/base/SimdUtil.h

```diff
@@ -429,16 +415,9 @@ struct Vectors<int16_t> {
     return (TV)_mm256_cmpgt_epi16(to256i(left), to256i(right));
   }
 
-  static auto compareResult(TV wide) {
-    return _mm256_movemask_epi8(to256i(wide));
-  }
-
-  static auto compareResult(__m256i wide) {
-    return _mm256_movemask_epi8(wide);
-  }
-
-  static auto compareBitMask(int32_t compareResult) {
-    return _pext_u32(compareResult, 0xaaaaaaaa);
+  static auto compareBitMask(TV wide) {
```

Comment:
could you add the comment you had on the PR description on why this uses a different approach?